test/test_unary_ufuncs.py

```diff
@@ -469,30 +468,6 @@ def test_batch_vs_slicing(self, device, dtype, op):
 
         self.assertEqual(actual, expected)
 
-    def _test_out_arg(self, op, input, output, expected):
-        if op.safe_casts_outputs:
```

Comment:
This is the only place where safe_casts_outputs is used.

Reply:
You're right. It was too aggressive to remove this test. I've added it back.